src/canonmn_int.cpp

```diff
@@ -2022,6 +2069,19 @@ namespace Exiv2 {
         return os;
     }
 
+    std::ostream& CanonMakerNote::printSi0x0001(std::ostream& os,
```

Comment:
[style] The style here does not seem to meet the style of the project (defined in the .clang-format.optional) file in the root level of the repository.

Being more specific, we allow the usage of up to 120 characters in one line, so it looks like the function and its arguments could go in 1 single line. The same happens in the **if** statement in lines 2077 & 2078.